packages/core/src/index.ts

```diff
@@ -183,8 +183,9 @@ export async function Auth(
     if (error instanceof CredentialsSignin) params.set("code", error.code)
 
     const pageKind = (isAuthError && error.kind) || "error"
-    const pagePath = config.pages?.[pageKind] ?? `/${pageKind.toLowerCase()}`
-    const url = `${internalRequest.url.origin}${config.basePath}${pagePath}?${params}`
```

Comment:
If we customize the `pages`, it should ignore the `basePath` (`/api/auth`) as well.